ReactNativeClient/android/settings.gradle

```diff
@@ -1,4 +1,6 @@
 rootProject.name = 'Joplin'
+include ':react-native-quick-actions'
+project(':react-native-quick-actions').projectDir = new File(rootProject.projectDir, '../node_modules/react-native-quick-actions/android')
```

Comment:
Note to reviewer: I didn't touch this file; these lines were added automatically when I installed `react-native-quick-actions` via npm and linked the project in xcode (as described in the OP of this PR)